packages/components/fields/search-select-field/src/search-select-field.tsx

```diff
@@ -1,320 +1,380 @@
    */
-  onInputChange: PropTypes.func,
+  onInputChange?: ReactSelectAsyncProps['onInputChange'];
   /**
    * Select the currently focused option when the user presses tab
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
    */
-  tabSelectsValue: PropTypes.bool,
+  tabSelectsValue?: ReactSelectAsyncProps['tabSelectsValue'];
   /**
    * Function that returns a promise, which is the set of options to be used once the promise resolves.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
    */
-  loadOptions: PropTypes.func.isRequired,
+  loadOptions: ReactSelectAsyncProps['loadOptions'];
   /**
    * The text shown while the options are being loaded
    */
-  loadingMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+  loadingMessage?: string | (() => string);
   /**
    * If cacheOptions is truthy, then the loaded data will be cached. The cache will remain until cacheOptions changes value.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
    */
-  cacheOptions: PropTypes.any,
+  cacheOptions?: ReactSelectAsyncProps['cacheOptions'];
   /**
    * Custom method to filter whether an option should be displayed in the menu
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
    */
-  filterOption: PropTypes.func,
+  filterOption?: ReactSelectAsyncProps['filterOption'];
   /**
    * The style of the an option in the dropdown menu. It could be single lined option or an option with more and custom info
    */
-  optionType: PropTypes.oneOf([
-    'single-property',
-    'double-property',
-    'multiple-properties',
-  ]),
+  optionType?: 'single-property' | 'double-property' | 'multiple-properties';
   /**
    * A map of errors. Error messages for known errors are rendered automatically.
    * <br />
    * Unknown errors will be forwarded to `renderError`
    */
-  errors: PropTypes.shape({
-    missing: PropTypes.bool,
-  }),
+  errors?: TFieldError;
   /**
    * Called with custom errors. This function can return a message which will be wrapped in an ErrorMessage. It can also return null to show no error.
-   * <br />
-   * Signature: `(key, error) => React.node`
    */
-  renderError: PropTypes.func,
+  renderError?: TErrorRenderer;
   /**
    * Indicates if the value is required. Shows an the "required asterisk" if so.
    */
-  isRequired: PropTypes.bool,
+  isRequired?: boolean;
   /**
    * Indicates whether the field was touched. Errors will only be shown when the field was touched.
    */
-  touched: (props, ...rest) =>
-    props.isMulti
-      ? PropTypes.arrayOf(PropTypes.bool, ...rest)(props, ...rest)
-      : PropTypes.bool(props, ...rest),
+  touched?: boolean[] | boolean;
   // LabelField
   /**
    * Title of the label
    */
-  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+  title: string | ReactNode;
```

Comment:
```suggestion
  title: ReactNode;
```
`ReactNode` already includes the `string` type.

![image](https://user-images.githubusercontent.com/97907068/154026687-88dbe93b-aed6-4539-ab20-7ac6e74fd425.png)